modules/jenkins-plugin/src/main/resources/com/smartbear/ready/jenkins/RemoteTestStarter/config.jelly

```diff
@@ -0,0 +1,81 @@
+                 description="The environment to use when running the project.">
+            <f:textbox/>
+        </f:entry>
+        <f:entry title="Remote repository" field="repositoryName"
+                 description="Repository on the TestServer where the project is stored.">
+            <f:textbox/>
+        </f:entry>
+    </f:section>
+    <f:section title="Connection Options">
+        <f:description>Only edit these fields if you want to override the default connection settings of the plugin.
+        </f:description>
+        <f:entry title="Server URL" field="serverUrl"
+                 description="The URL to the root of the TestServer, e.g. https://our.testserver.instance:8443/v1.">
+            <f:textbox default="${descriptor.serverUrl}"/>
+        </f:entry>
+        <f:entry title="Username" field="userName"
+                 description="The username on the TestServer instance.">
+            <f:textbox default="${descriptor.userName}"/>
+        </f:entry>
+        <f:entry title="Server password" field="password"
+                 description="The password to use for logging in to the TestServer instance.">
+            <f:password/>
+        </f:entry>
+        <f:validateButton
+                title="${%Test Connection}" progress="${%Testing...}"
+                method="testServerConnection" with="serverUrl,userName,password"/>
+    </f:section>
+    <f:advanced>
+        <f:section title="Advanced Test Options">
+            <f:entry title="Extra files" field="extraFiles"
+                     description="A path, or a comma-separated list of paths, to file(s) that should be uploaded along with the project or recipe.">
+                <f:textbox/>
+            </f:entry>
+            <f:entry title="Project Properties" field="projectProperties"
+                     description="One per line in the form name=value">
+                <f:textarea name="projectProperties"/>
+            </f:entry>
+            <f:section title="XML project options">
+                <f:description>These fields are only applicable for XML/Composite projects</f:description>
+                <f:entry title="Project file password" field="projectFilePassword"
+                         description="The password used to decrypt the project file (leave empty if it is not encrypted).">
+                    <f:password/>
+                </f:entry>
+                <f:entry title="Endpoint" field="hostAndPort"
+                         description="A string in the format host:port, which will replace the host and port in all HTTP requests sent by the test.">
+                    <f:textbox/>
+                </f:entry>
+                <f:entry title="Tags" field="tags"
+                         description="A comma separated list of tags to filter test cases by.">
+                    <f:textbox/>
+                </f:entry>
+                <f:entry title="Report file" field="reportFile"
```

Comment:
isn't there a jenkins convention here that needs to be followed so Jenkins will automatically include the results in  it own test reports? or is this separate?